packages/core/admin/shared/contracts/audit-logs.ts

```diff
@@ -0,0 +1,43 @@
+import { errors } from '@strapi/utils';
+import { Entity, SanitizedAdminUser } from './shared';
+import { Schema } from '@strapi/types';
+
+interface CollectionAuditLog extends Schema.CollectionType {
+  user?: SanitizedAdminUser;
+}
+
+interface SingleAuditLog extends Schema.SingleType {
+  user?: SanitizedAdminUser;
+}
+
+type AuditLog = CollectionAuditLog | SingleAuditLog;
+
+namespace GetAll {
+  export interface Request {
+    body: {};
+    query: {};
+  }
+
+  export interface Response {
+    data: AuditLog[];
+    error?: errors.ApplicationError;
+  }
```

Comment:
This doesn't seem to match what I see when calling the endpoint, which is more like

```ts
interface Response {
  pagination: {
    page: number;
    pageSize: number;
    pageCount: number;
    total: number;
  };
  results: AuditLog[];
}
```

error is probably right though I haven't tested